kolibri/plugins/learn/assets/src/views/SearchPage.vue

```diff
@@ -43,9 +55,10 @@
     $trs: {
       searchPageHeader: 'Search',
       noSearch: 'Search by typing in the box above',
-      showingResultsFor: "Results for '{searchTerm}'",
+      showingResultsFor: "Results for '{searchTerm}' ({results} of {totalResults})",
```

Comment:
We should not show the `({results} of {totalResults})` part of this string when there are no results. Also would this part of the string be better off being on it's own below the heading? Also, maybe `1-30 of 70` instead of `30 of 70`?

Reply:
We don't show this string when there are no results, there is a separate `noResults` string that gets displayed instead, so think this should be fine as is.

Reply:
The heading is displaced as long there is a search query.

Reply:
Oh, sorry, misread the template logic. We should probably make this a v-else block.

Reply:
Have updated.

Reply:
Have updated.